spec/types/calculation.md

```diff
@@ -341,3 +347,32 @@ To evaluate a `CalcValue` production `value` into a `CalculationValue` object:
   > Allowing variables to return unquoted strings here supports referential
   > transparency, so that `$var: fn(); calc($var)` works the same as
   > `calc(fn())`.
+
+### `ParenthesizedVar`
+
+> If a `var()` is written directly within parentheses, it's necessary to
+> preserve those parentheses. CSS resolves `var()` by literally replacing the
+> function with the value of the variable and *then* parsing the surrounding
+> context.
+>
+> For example, if `--ratio: 2/3`, `calc(1 / (var(--ratio)))` is parsed as
+> `calc(1 / (2/3)) = calc(3/2)` but `calc(1 / var(--ratio))` is parsed as
+> `calc(1 / 2/3) = calc(2/3)`.
```

Comment:
should be `= calc(1/6)`

Reply:
Thanks! https://github.com/sass/sass/pull/3157